test/test_det_noise_sim.cpp

```diff
@@ -38,7 +38,7 @@ std::unique_ptr<qc::QuantumComputation> detGetAdder4Circuit() {
 TEST(DeterministicNoiseSimTest, MeasurementOne) {
     auto quantumComputation = std::make_unique<qc::QuantumComputation>(2);
     quantumComputation->emplace_back<qc::StandardOperation>(2, 0, qc::H);
-    quantumComputation->emplace_back<qc::StandardOperation>(2, dd::Controls{dd::Control{0}}, 1, qc::X);
+    quantumComputation->emplace_back<qc::StandardOperation>(2, qc::Controls{qc::Control{0}}, 1, qc::X);
     quantumComputation->emplace_back<qc::NonUnitaryOperation>(2, 0, qc::Measure);
```

Comment:
Would look cleaner here to just use the convenience functions we expose in the QFR.
Also `qc::Controls{qc::Control{0}}` could just be `0_pc`

This also holds for some occurrences below.